src/audio/midi/message.rs

```diff
@@ -0,0 +1,21 @@
+use crate::Result;
+use std::convert::TryFrom;
+use wmidi::MidiMessage;
+
+#[derive(Debug)]
+pub struct Message {
+    pub timestamp: u64,
```

Comment:
I need the timestamp, which isn't included in the actual MIDI message, so I wrapped up the messages in this struct.